domain/src/Entity/Domain.php

```diff
@@ -474,4 +482,12 @@ public function getMatchType() {
     return $this->matchType;
   }
 
+  public function getHomepage() {
```

Comment:
here we also need to include the docblocks.  adding {@inheritdoc} would be enough (if you document the interface)